examples/nek5000/bp1.usr

```diff
@@ -955,27 +961,28 @@ c     Helmholtz equations
       real ap1(lx,lelt)
       real  p1(lx,lelt)
       real  h1(lx,lelt), h2(lx,lelt)
-      integer ceed,op_mass,vec_ap1,vec_p1,vec_qdata
-      integer*8 offset
+      integer ceed,op_mass,vec_ap1,vec_p1,vec_qdata,err
+      integer i,e
 
       call ceedvectorsetarray(vec_p1,ceed_mem_host,ceed_use_pointer,
      $  p1,err)
+      call ceedvectorsetarray(vec_ap1,ceed_mem_host,ceed_use_pointer,
+     $  ap1,err)
+
       call ceedoperatorapply(op_mass,vec_p1,vec_ap1,
      $  ceed_request_immediate,err)
-      call ceedvectorgetarray(vec_ap1,ceed_mem_host,ap1,offset,err)
```

Comment:
Have you had a chance to check on GPU yet? @jedbrown and I were talking, and the library likely needs a function such as `CeedVectorSync` to force a device to host update in situations like this.

Reply:
Yes, I agree. Otherwise, I don't think this code will run on the GPU.

Reply:
I think it makes sense to put this in a later PR. I'll make an issue for this frontend feature.